app/views/employers/plan_years/plan_selection/_plan_details.html.erb

```diff
@@ -21,8 +21,9 @@
     </header>
   </section>
   <section class="col-xs-6">
-    <h4 class="change-dental-reference-plan pull-right" style="display: none;">Change Reference Plan?</h4>
-    <% render partial: "employers/plan_years/dental_reference_plans_options_modal" %>
+    <h4 class="change-health-reference-plan pull-right" style="display: none;">Change Reference Plan?</h4>
+    <!-- TODO: WHAT WAS HERE? This was referencing a dental modal which can't possibly be correct ~ -->
```

Comment:
Actually I think this might have been right-ish.  As per usual, the dental is not done in the same way as the normal one.  I *think* this is actually where in the UI the dental stuff is supposed to go.